src/ui/components/Difference/index.jsx

```diff
@@ -2,40 +2,130 @@ import React, { useState, useEffect } from "react";
 import styled from "styled-components";
 import { nanoid } from "nanoid";
 
+import { useNavigate } from "react-router-dom";
 import Description from "../shared/Description";
 import Button from "../shared/Button";
-import Modal from "../shared/Modal";
+import ToastPopup from "../shared/Toast";
 import Loading from "../shared/Loading";
+import Modal from "../shared/Modal";
 import Popup from "../shared/Popup";
 
+import usePageListStore from "../../../store/projectPage";
+import useProjectStore from "../../../store/project";
+
+import isOwnProperty from "../../../utils/isOwnProperty";
+import postMessage from "../../../utils/postMessage";
+import getDiffingResultQuery from "../../../services/getDiffingResultQuery";
 import processDifferences from "../../../utils/processDifferences";
```

Comment:
- `P1`
```suggestion
import isOwnProperty from "../../../utils/isOwnProperty";
import postMessage from "../../../utils/postMessage";
import processDifferences from "../../../utils/processDifferences";
import getDiffingResultQuery from "../../../services/getDiffingResultQuery";
```

`utils`끼리 모아놓으면 가독성에 더 도움이 될 것 같습니다!